src/rust/plugin-bootstrap/plugin-bootstrap-init.service

```diff
@@ -0,0 +1,40 @@
+# Contents of /etc/systemd/system/plugin-bootstrap-init.service
+[Unit]
+Description=Plugin Init
+After=network.target
+
+[Service]
+Type=oneshot
+ExecStart=/usr/local/bin/init-grapl-plugin
+User=root
+Group=root
```

Comment:
Seems a little weird to run as root then set sandboxing stuff below.

Looks like the motivation for using root is to have read access to `/etc/ssl/private/plugin-client-cert.pem`, is that right?

Reply:
Yeah, it's in order to have the right file access. I agree that root is overly permissive here, but I figure that we can create a more restricted user in the future.

I *assume* that root here essentially means that escaping the sandboxing is possible, but I don't actually know.